Sources/Commands/SwiftTestCommand.swift

```diff
@@ -339,11 +334,6 @@ public struct SwiftTestCommand: AsyncSwiftCommand {
                 .filteredTests(specifier: options.testCaseSpecifier)
                 .skippedTests(specifier: options.skippedTests(fileSystem: swiftCommandState.fileSystem))
 
-            // If there were no matches, emit a warning.
-            if tests.isEmpty {
-                swiftCommandState.observabilityScope.emit(.noMatchingTests)
```

Comment:
This diagnostic is misleading if Swift Testing _does_ have matches. I propose punting on it for now until we have a good way to correlate results from both libraries.